vaadin-checkbox-flow-parent/vaadin-checkbox-flow-integration-tests/src/test/java/com/vaadin/flow/component/checkbox/tests/validation/BasicValidationIT.java

```diff
@@ -69,6 +69,17 @@ public void webComponentCanNotModifyInvalidState() {
         assertWebComponentCanNotModifyInvalidState();
     }
 
+    @Test
+    public void clientSideInvalidStateIsNotPropagatedToServer() {
```

Comment:
Can also add this for `RadioButtonGroup`.

Reply:
Right, done.